models/question.js

```diff
@@ -0,0 +1,26 @@
+var mongoose = require('mongoose');
+
+const questionSchema = mongoose.Schema({
+    question: {
+        type: String,
+        required: true
+    },
+    answers: [{
+        answer: {
```

Comment:
Since we have an array of answer objects here we don't need the additional "answer" key.
```suggestion
```